### DISCUSSION TITLE
Pass command line arguments to program when using `poac run`

### DISCUSSION THREAD

**Discussion by oraqlle** [cabinpkg/cabin#788] [Q&A] [answered]
Does Poac currently support passing command line arguments to a program when using the `poac run` command?

**Comment by ken-matsui** [comment, marked as answer]
This was implemented with 0.7.0. e.g., https://github.com/poac-dev/poac/blob/766bbe3c5c74bfa6020308d65004a3a38658af8e/.github/workflows/linux.yml#L63
